sqiffy/src/main/kotlin/com/dzikoysk/sqiffy/changelog/generator/dialects/GenericSqlSchemeGenerator.kt

```diff
@@ -82,16 +119,30 @@ abstract class GenericSqlSchemeGenerator : SqlSchemeGenerator {
                 FLOAT -> "FLOAT"
                 DOUBLE -> "DOUBLE"
                 DATE -> "DATE"
-                TIMESTAMP -> "TIMESTAMP"
                 else -> throw UnsupportedOperationException("Cannot create data type based on $property")
             }
         }
 
     protected open fun createDataTypeWithAttributes(property: PropertyData, availableEnums: Enums): String =
         createDataType(property, availableEnums).let {
-            if (!property.nullable) "$it NOT NULL"
-            else it
+            var result: String =
+                property.default?.let { default -> "$it DEFAULT ${default.toSqlDefault(property)}" } ?: it
+            result = if (!property.nullable) "$result NOT NULL" else it
+            result
+        }
+
+    private fun String.toSqlDefault(property: PropertyData): String {
+        return createSqlDefault(this, property) ?: createRegularDefault(this, property)
+    }
+
+    abstract fun createSqlDefault(rawDefault: String, property: PropertyData, dataType: DataType = property.type!!): String?
+
+    private fun createRegularDefault(rawDefault: String, property: PropertyData, dataType: DataType = property.type!!): String {
+        return when (dataType) {
+            UUID_TYPE, ENUM, CHAR, VARCHAR, TEXT -> "'$rawDefault'"
+            else -> rawDefault
```

Comment:
I think it'd be healthier for now to throw an exception for unknown dataType default in `toSqlDefault` once the createSqlDefault & createRegularDefault return `null`